integrations/nvidia/tests/test_base_url.py

```diff
@@ -60,6 +61,16 @@ def test_base_url_valid_generator(base_url: str) -> None:
         "http://localhost:8888/chat/completions",
     ],
 )
+# todo: add this to test_base_url_invalid_not_hosted
```

Comment:
What is this about?

Reply:
a note to myself, which i decided to skip because it was a refactor. i'll remove it.